docs-src/zdm-core/modules/migrate/partials/interactive-lab.adoc

```diff
@@ -1,3 +1,5 @@
+Now that you've seen a conceptual overview of the process, why not putting what you learned in practice? 
```

Comment:
```suggestion
Now that you've seen a conceptual overview of the process, why not put what you learned into practice? 
```

Other options:
- "Now that you've seen a conceptual overview of the process, let's put what you learned into practice."
- "Now that you've seen a conceptual overview of the process, you can try putting what you learned into practice."

Reply:
Done in latest commit, pushed, and rebuilt on coppi. Thanks Eric.